frontend/src/components/CheckboxGroup/CheckboxGroupValidator.tsx

```diff
@@ -0,0 +1,221 @@
+import { List } from 'immutable';
+import React, { FunctionComponent, useState } from 'react';
+import { Filters, OperationStepMap } from '../../types/operations';
+import { BasicModal } from '../BasicModal';
+
+interface ComponentProps {
+  steps?: List<OperationStepMap>;
+}
+
+const CheckboxGroupValidator: FunctionComponent<ComponentProps> = (props) => {
```

Comment:
Given the previous comments, I'm sure you won't be surprised that I'm asking you to move this component to its own folder, and rename it to something not related to the `CheckboxGroup`. Name it something related to the `select` step since it'll be associated with it... e.g. `SelectColumnValidator`

Reply:
You'll also need to have the active step as a prop, cos your current implementation doesn't take into account that a `select` step can come AFTER a filter. So it checks all filters, even those not affected by the changes in the current select. Let me know if you need more clarification on this.

Reply:
Renamed and added active step prop